moped-database/migrations/1675978843251_alter_table_public_moped_proj_components_add_column_phase_id/down.sql

```diff
@@ -0,0 +1,5 @@
+-- drop all new columns (this will also drop constraints)
+alter table "public"."moped_proj_components" drop column "phase_id";
+alter table "public"."moped_proj_components" drop column "subphase_id";
+alter table "public"."moped_proj_components" drop column "phase_end";
+alter table "public"."moped_proj_components" drop column "date_added";
```

Comment:
The least important thing in the world, but if you do end up editing these files again, you may want to drop a new line at the end of this file.

Reply:
thanks @frankhereford ! added the newline